api/client.go

```diff
@@ -172,10 +172,6 @@ func (c *Client) stream(ctx context.Context, method, path string, data any, fn f
 			return fmt.Errorf("unmarshal: %w", err)
 		}
 
-		if errorResponse.Error != "" {
```

Comment:
Moved since StatusError should have the error message set, so Im not sure it was reachable.

Reply:
This may occur if there's an error in the middle of the stream

Reply:
The check is done after the status code check, possible to show a test-case that would demonstrate this or is it a pain where it's a mid-stream thing?